scanners/zap-extended/scanner/setup.py

```diff
@@ -0,0 +1,26 @@
+import setuptools
```

Comment:
Do we want to package a wheel and push it to PyPI? If not, we can omit this file

Reply:
Good point 👍🏻 As for now we have no plans to publish this module somewhere else.